src/utils/cpg.ts

```diff
@@ -74,64 +75,107 @@ export class CPGExporter {
     this.criteria = criteria;
     this.nestedBranch = [];
     this.activityDefinitions = {};
-    this.libraries = this.createLibraries();
+    this.pathways = pathways;
   }
 
   export(): Bundle {
-    const library = this.libraries[0];
-    const cpgRecommendation = this.createPlanDefinition(
+    const strategyDefinition = this.createPlanDefinition(
       uuidv4(),
-      this.pathway.name,
-      this.pathway.description,
-      'recommendation',
-      library.id
+      `Strategy${this.pathway.name}`,
+      `Strategy for ${this.pathway.name}`,
+      'strategy'
     );
 
-    /*
+    this._export(strategyDefinition, this.pathway);
```

Comment:
`strategyDefinition` could probably be a class attribute if you want so it doesn't need to be passed here. I'm also fine with the way it is now if you prefer

Reply:
im partial to the way it is now, I think it makes the functions more modular